apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-modal/index.tsx

```diff
@@ -27,29 +28,31 @@ const LaunchModal: React.FunctionComponent< Props > = ( {
 	onSubmit,
 	step,
 	isLaunching,
-} ) => {
-	const handleClose = () => {
-		onClose?.();
-	};
+} ) => (
+	<Modal
+		className="nux-launch-modal"
+		overlayClassName="nux-launch-modal-overlay"
+		bodyOpenClassName="has-nux-launch-modal"
+		onRequestClose={ () => onClose?.() }
+		title=""
+	>
+		<div className="nux-launch-modal-header">
+			<div className="nux-launch-modal-header__wp-logo">
+				<Icon icon={ wordpress } size={ 36 } />
+			</div>
 
-	return (
-		<Modal
-			className="nux-launch-modal"
-			overlayClassName="nux-launch-modal-overlay"
-			bodyOpenClassName="has-nux-launch-modal"
-			onRequestClose={ handleClose }
-			title={
-				isLaunching
-					? __( 'Hooray! Your site will be ready shortly.', 'full-site-editing' )
-					: __(
-							"You're almost there! Review a few things before launching your site!",
-							'full-site-editing'
-					  )
-			}
-		>
-			{ isLaunching ? 'launch animation' : <Launch step={ step } onSubmit={ onSubmit }></Launch> }
-		</Modal>
-	);
-};
+			<Button
+				isLink
+				className="nux-launch-modal-header__close-button"
+				onClick={ onClose }
+				aria-label={ __( 'Close dialog', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **123** times:
`translate( 'Close dialog' )` *ES Score: 14*